src/plugins/cache.js

```diff
@@ -107,6 +127,21 @@ export default class Cache {
 
   async fetchManagers (...managers: Manager[]): Promise<Plugin[]> {
     let plugins = []
+    if (this.cache.node_version !== process.version) {
+      let lock = new Lock(this.out)
+      let handled = true
+
+      let downgrade = await lock.upgrade()
+      for (let manager of managers) {
+        handled = await manager.handleNodeVersionChange() && handled
```

Comment:
@dickeyxxx one impediment to storing it for individual linked plugins (at least in the `linked_plugins.json`) is that `plugins` is an array of strings which if it were objects would be the natural place to store the version.  The other option would be to instead store the `node_version` in the `plugins.json` cache where I would bulk update the user plugins and the individual linked plugins.  Thoughts?
```
{
  "version": "1",
  "updated_at": "2017-06-26T18:50:19.278Z",
  "plugins": [
    "/Users/rbriggs/src/github.com/heroku/heroku-apps"
  ]
}
```